fire/cli/info.py

```diff
@@ -341,6 +350,13 @@ def punkt_fuld_rapport(
     default=False,
     help="Udskriv også sjældent anvendte elementer",
 )
+@click.option(
+    "-H",
+    "--historik",
```

Comment:
Er -v/--verbose optaget? Eller skal vi holde os helt fra engelske udtryk, selvom de er normale i mange andre kommandolinje-værktøjer?

Vi kunne måske sige, at -v betyder, vis mere:)

Reply:
Vi har følgende muligheder med denne PR:
```
Options:
  -K, --koord TEXT     ts: Udskriv også tidsserier; alle: Udskriv også
                       historiske koordinater; ingen: Udelad alle
  -O, --obs TEXT       niv/alle: Udskriv observationer
  -D, --detaljeret     Udskriv også sjældent anvendte elementer
  -H, --historik       Udskriv også ikke-gældende elementer
  -n, --antal INTEGER  Begræns antallet af punkter der udskrives
  --db [prod|test]     Vælg en specifik databaseforbindelse -
                       default_connection i fire.ini bruges hvis intet vælges.
  -m, --monokrom       Vis ikke farver i terminalen
  --debug              Vis debug output fra FIRE-databasen.
  --help               Vis denne hjælp tekst
```

`-K`, `-O`, `-D`,  og `-H` går hånd i hånd og giver mulighed for at fintune output til ens behov. Jeg kunne godt tænke mig en klassisk `-v`/`--verbose` på et tidspunkt som kobles på en logger, hvor man kan vælge forskellige niveauer af snakkesalighed. Men det forestiller jeg mig mere som den klassiske løsning med warning, info, debug osv.

Reply:
Godt argument. Og tak for rationalet:)